libnavui-androidauto/api/current.txt

```diff
@@ -275,7 +275,6 @@ package com.mapbox.androidauto.car.location {
 
   public final class CarLocationRenderer implements com.mapbox.maps.extension.androidauto.MapboxCarMapObserver {
     ctor public CarLocationRenderer(com.mapbox.androidauto.car.MainCarContext mainCarContext);
-    method public void onAttached(com.mapbox.maps.extension.androidauto.MapboxCarMapSurface mapboxCarMapSurface);
```

Comment:
Explanation for this is here https://github.com/mapbox/mapbox-navigation-android/pull/6396